crates/aranya-client/tests/tests.rs

```diff
@@ -613,13 +621,14 @@ async fn test_afc_one_way_two_chans() -> Result<()> {
     let got = team
         .memberb
         .client
-        .try_recv_afc_data()
+        .afc()
+        .try_recv_data()
         .expect("should have a message");
-    let want = AfcMsg {
+    let want = Message {
         data: msgs[0].as_bytes().to_vec(),
         // We don't know the address of outgoing connections, so
         // assume `got.addr` is correct here.
-        addr: got.addr,
+        address: got.address,
```

Comment:
Switch back to using `addr` instead of `address` for public APIs and internal code:
```suggestion
        addr: got.addr,
```

Reply:
This is part of Message which is a public type as we talked about